packages/running/style/base.css

```diff
@@ -7,6 +7,9 @@
 | Variables
 |----------------------------------------------------------------------------*/
 
+/* Import toolkit styles */
+@import url('../../ui-components/style/toolkit.css');
+
```

Comment:
As this is included in another package (i.e. ui-components) on which this package depends. You should not import it here. But you need to imported it in `packages/ui-components/style/base.css` by adding:

```css
@import './toolkit.css';
```